harness/determined/pytorch/_trainer.py

```diff
@@ -72,6 +72,7 @@ def fit(
         latest_checkpoint: Optional[str] = None,
         step_zero_validation: bool = False,
         test_mode: bool = False,
+        enable_tensorboard_logging: bool = True,
```

Comment:
if this is on the context, it might be a better fit for `_trainer.init`?

Reply:
I don't have a strong opinion here, so happy to go either way.

Following are my original reason to put it under fit:
If we look at the arguments under init vs fit, I just think this new option seem more similar with arguments of fit vs init (e.g. step_zero_validation, test_mode) whereas arguments on init are more like exp_config, hparams, etc.

Reply:
i'm leaning towards putting it in the context's `init` because it seems like this value is specific to the "runtime context" of the training loop, and doesn't really have to do with the actual `.fit()` training loop configurations. feels like it belongs more with `debug_enabled`, `managed_training` in the context (though currently not configurable/passed in by the user), rather than training loop arguments like reporting/validation steps. 

i feel it's also easier to tell where/when the context values are being set if we try and keep them in the same place.